robot.py

```diff
@@ -16,19 +16,20 @@ class Robot(MagicRobot):
     flipper = Flipper
 
     def createObjects(self):
-        """ 
-        f = front, b = back. r = right, l = left
-        """
-        self.fl_motor = CANTalon(0)
-        self.fr_motor = CANTalon(1)
-        self.bl_motor = CANTalon(2)
-        self.br_motor = CANTalon(3)
-
-        self.robot_drive = wpilib.RobotDrive(
-            self.fl_motor,
-            self.fr_motor,
-            self.bl_motor,
-            self.br_motor)
+        self.front_left_motor = CANTalon(1)
+        self.back_left_motor = CANTalon(2)
+        self.front_right_motor = CANTalon(3)
+        self.back_right_motor = CANTalon(4)
+
+        self.back_left_motor.setControlMode(CANTalon.ControlMode.Follower)
+        self.back_right_motor.setControlMode(CANTalon.ControlMode.Follower)
+
+        self.back_left_motor.set(1)
```

Comment:
Would this be better as `self.back_left_motor.set(self.front_left_motor.getDeviceID())`?

Reply:
Yes.